src/deployment/AggregateDeployment.s.sol

```diff
@@ -192,6 +192,122 @@ contract AggregateDeployment is BaseDeployment {
         return dataProvider;
     }
 
+    function deployNew() public {
+        // This is performed from a deployer without rights to list.
+        bytes[] memory deployData = new bytes[](9);
+
+        emit log("--- Liquity 275% CR and 400% CR deployments ---");
+        {
+            LiquityTroveDeployment liquityTroveDeployment = new LiquityTroveDeployment();
+            liquityTroveDeployment.setUp();
+
+            // Deploy the bridges
+            address tb275 = liquityTroveDeployment.deploy(275);
+            address tb400 = liquityTroveDeployment.deploy(400);
+
+            // Open the troves
+            liquityTroveDeployment.openTrove(tb275);
+            liquityTroveDeployment.openTrove(tb400);
```

Comment:
These 2 actions will require quite a lot of ETH as collateral. Might be better to do it in separate txs.